docs/source/models/gbd2017/causes/neonatal_sepsis/index.rst

```diff
@@ -13,8 +13,92 @@ Case definition in GBD 2017
 Neonatal sepsis and other neonatal infections are infections during the neonatal
 period that advance to a systemic bloodstream infection, the underlying cause of
 which can be meningitis, gastroenteritis, or other aetiologies. Neonatal
-pneumonia, however, is not included -- it is captured in our modelling of
+pneumonia, however, is not included -- it is captured in the modelling of
 pneumonia as a separate entity.
 
 .. todo::
-   Describe cause model
+   Add more information and references. In particular, find data about differences in global and 
+   gbd definitions, global prevalence and relation to folic acid during pregnancy.
+
+Modeling Measles in GBD 2017
+----------------------------
+
+It is worth noting that the ICD 10 and ICD 9 codes used in non-fatal and fatal GBD models are different. 
+The ICD 10 codes used in non fatal model are P36-P36.9, P38-P39.9. K29.0-K29.00, K29.1-K29.20, P77-P78.1 
```

Comment:
```suggestion
The ICD 10 codes used in non fatal model are P36-P36.9, P38-P39.9. The codes K29.0-K29.00, K29.1-K29.20, P77-P78.1 
```
This was a bit hard to read with all the codes adjacent to each other. Editing this line and the next for clarity.

Reply:
Is K29.0-K29.00 a range? or ==K29.0?

Reply:
I am not sure. I am assuming its a range. I checked the ICD codes in the range. They seem relevant but couldn't find all of the codes in the range.

Reply:
K29.0 and K29.00 are different. K29.00 is "Acute gastritis without bleeding". K29.0 is "Acute gastritis"